packages/reactivity/src/computed.ts

```diff
@@ -15,14 +15,20 @@ export interface WritableComputedRef<T> extends Ref<T> {
   readonly effect: ReactiveEffect<T>
 }
 
-export type ComputedGetter<T> = (...args: any[]) => T
+export type ComputedGetter<T> = () => T
+export type ComputedGetterWithInstance<T> = (...args: any[]) => T
 export type ComputedSetter<T> = (v: T) => void
 
 export interface WritableComputedOptions<T> {
   get: ComputedGetter<T>
   set: ComputedSetter<T>
 }
 
+export interface WritableComputedOptionsWithVModel<T> {
```

Comment:
Sorry I wasn't more explicit but this needs to be renamed to `...WithInstance` for consistency